### PR TITLE
fix(sale calcs): Fix calculations with sale prices.

### PR DESCRIPTION
**Pull request by Camwyn**
Add a wrapper to the actual price (default or sale price) to better target for js
calculations/updates.

Great catch @jesseeproductions !

:ticket: [etp-47]

### CODE REVIEW

File: src/views/blocks/tickets/extra-price.php [hunk 1]
```diff
@@ -33,8 +33,10 @@ class="tribe-common-b2 tribe-common-b1--min-medium tribe-tickets__item__extra__p
 >
 	<?php if ( ! empty( $ticket->on_sale ) ) : ?>
 		<span class="tribe-common-b2 tribe-tickets__original_price">
-		<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->regular_price, $post_id, $provider_class ) ?>
+			<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->regular_price, $post_id, $provider_class ) ?>
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/extra-price.php]:
```suggestion
			<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->regular_price, $post_id, $provider_class ); ?>
```

File: src/views/blocks/tickets/extra-price.php [hunk 2]
```diff
@@ -33,8 +33,10 @@ class="tribe-common-b2 tribe-common-b1--min-medium tribe-tickets__item__extra__p
...
 		</span>
 	<?php endif; ?>
-	<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->price, $post_id, $provider_class ) ?>
+	<span class="tribe-tickets__sale_price">
+		<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->price, $post_id, $provider_class ) ?>
```
Comment (sc0ttkclark) [inline src/views/blocks/tickets/extra-price.php]:
```suggestion
		<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( $ticket->price, $post_id, $provider_class ); ?>
```

**Comment by sc0ttkclark** [comment]
What’s the common stuff here? Is that from B20.01?

**Comment by Camwyn** [comment]
> What’s the common stuff here? Is that from B20.01?

Yeah, it snuck in on me. Corrected.

**Pull request merged**